<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addBook</class>
 <widget class="QMainWindow" name="addBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(67, 67, 67);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(64, 64, 64);
font: 75 14pt &quot;Arial&quot;;
background-color: rgb(187, 134, 252);</string>
    </property>
    <property name="text">
     <string>   ADD BOOK</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>511</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(64, 64, 64);
font: 75 14pt &quot;Arial&quot;;
background-color: rgb(187, 134, 252);</string>
         </property>
         <property name="text">
          <string>   title   :    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_title">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="placeholderText">
          <string>(optional)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">color: rgb(64, 64, 64);
font: 75 14pt &quot;Arial&quot;;
background-color: rgb(187, 134, 252);</string>
         </property>
         <property name="text">
          <string>  author :  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_aouthor">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="placeholderText">
          <string>(optional)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">color: rgb(64, 64, 64);
font: 75 14pt &quot;Arial&quot;;
background-color: rgb(187, 134, 252);</string>
         </property>
         <property name="text">
          <string>  ISBN :    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_ISBN">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="placeholderText">
          <string>(Mandatory)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">color: rgb(64, 64, 64);
font: 75 14pt &quot;Arial&quot;;
background-color: rgb(187, 134, 252);</string>
         </property>
         <property name="text">
          <string>  price :    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_price">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="placeholderText">
          <string>(optional)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">color: rgb(64, 64, 64);
font: 75 14pt &quot;Arial&quot;;
background-color: rgb(187, 134, 252);</string>
         </property>
         <property name="text">
          <string> number : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_number">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="placeholderText">
          <string>(optional)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 218, 197);
color: rgb(15, 15, 15);</string>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>400</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 218, 197);
color: rgb(15, 15, 15);</string>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
